downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md

```diff
@@ -167,7 +167,7 @@ type Employee record {|
 
 ### Improvements related to JSON compatibility
 
-Three langlib functions were introduced and `toJsonString` is now allowed to have `anydata`.
+Three new langlib methods have been introduced to improve converting to and from JSON. Additionally, the `toJsonString` method can now be called on `anydata` values.
```

Comment:
We don't really use the term "langlib" in docs like these right?

Reply:
How about something like 

```suggestion
Three new methods have been introduced to the `ballerina/lang.value` module to facilitate converting to and from JSON. Additionally, the `toJsonString` method can now be called on `anydata` values.
```

Reply:
+1